tutorials/tts/FastPitch_Training_GermanTTS.ipynb

```diff
@@ -0,0 +1,1226 @@
+    "    ```python\n",
+    "    attn_prior = torch.from_numpy(\n",
+    "          beta_binomial_interpolator(spect_len.item(), text_len.item())\n",
+    "        ).unsqueeze(0).to(text.device)\n",
+    "    ```\n",
+    "3. We use both of them to generate synthetic mel spectrogram via `spec_model.forward` method:\n",
+    "    ```python\n",
+    "    spectrogram = spec_model.forward(\n",
+    "          text=text, \n",
+    "          input_lens=text_len, \n",
+    "          spec=spect, \n",
+    "          mel_lens=spect_len, \n",
+    "          attn_prior=attn_prior,\n",
+    "          speaker=speaker,\n",
+    "        )[0]\n",
+    "    ```\n",
+    "    \n",
+    "Repeat the above script for train and validation datasets as well. \n",
+    "\n",
+    "Finally, the `DataGermanTTS/synmels` will look like:\n",
+    "```\n",
+    "DataGermanTTS/synmels/:\n",
+    "hifigan_test_ft.json\n",
+    "hifigan_train_ft.json\n",
+    "hifigan_val_ft.json\n",
+    "test\n",
+    "train\n",
+    "val\n",
+    "\n",
+    "DataGermanTTS/synmels/test:\n",
+    "mel_0.npy\n",
+    "mel_1.npy\n",
+    "...\n",
+    "\n",
+    "DataGermanTTS/synmels/train:\n",
+    "mel_0.npy\n",
+    "mel_1.npy\n",
+    "...\n",
+    "\n",
+    "DataGermanTTS/synmels/val:\n",
+    "mel_0.npy\n",
+    "mel_1.npy\n",
+    "...\n",
+    "```\n",
+    "\n",
+    "Example HiFi-GAN manifest:\n",
+    "```json\n",
+    "{\"audio_filepath\": \"DataGermanTTS/thorsten-de/wavs/e50eb02c25353f85549900d2fc1e0e32.wav\", \"duration\": 2.409977, \"text\": \"Geht die Schandtat auf sein Konto?\", \"normalized_text\": \"Geht die Schandtat auf sein Konto?\", \"mel_filepath\": \"DataGermanTTS/synmels/test/mel_0.npy\"}\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fe160e99",
+   "metadata": {},
+   "source": [
+    "## Launch finetuning\n",
+    "\n",
+    "Download the pre-trained HiFi-GAN from NGC."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "89550a07",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!(cd DataGermanTTS && \\\n",
+    "  wget --content-disposition https://api.ngc.nvidia.com/v2/models/nvidia/nemo/tts_hifigan/versions/1.0.0rc1/zip -O tts_hifigan_1.0.0rc1.zip && \\\n",
+    "  unzip tts_hifigan_1.0.0rc1.zip)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5be984be",
+   "metadata": {},
+   "source": [
+    "We will be re-using the existing HiFi-GAN config and HiFi-GAN pretrained on English."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "990761d0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!(cd NeMoGermanTTS && \\\n",
+    "  python hifigan_finetune.py --config-path NeMoGermanTTS --config-name hifigan.yaml \\\n",
+    "    model.max_steps=10 \\\n",
+    "    model.optim.lr=0.00001 \\\n",
+    "    model.train_ds=train_ds_finetune \\\n",
+    "    model.validation_ds=val_ds_finetune \\\n",
```

Comment:
I get the below error unless I change the "." to "/" in the arguments. Is the behavior here inconsistent?

```
...
    raise ValueError(f"No dataset for {name}")
ValueError: No dataset for train
```

Reply:
corrected